data/example_bmi_multi_realization_config.json

```diff
@@ -60,16 +60,10 @@
                                 "registration_function": "register_bmi_cfe",
                                 "variables_names_map": {
                                     "water_potential_evaporation_flux": "ETRAN",
```

Comment:
Here it is 
` "water_potential_evaporation_flux": "ETRAN",`

however, in `example_bmi_multi_realization_config_w_netcdf.json`, it is
`"water_potential_evaporation_flux": "EVAPOTRANS",`

it should be `EVAPOTRANS` as well.